sips/pending/_posts/2017-02-07-make-types-behave-like-expressions.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: sip
+disqus: true
+title: SIP-NN - Make types behave like expressions 
```

Comment:
I think the current title is too general, it's not clear what is this sip about if you only read the title.
May I suggest:
`Types have precedence and associativity rules as expressions`